cohortextractor/backends/graphnet.py

```diff
@@ -5,11 +5,69 @@
 class GraphnetBackend(BaseBackend):
     backend_id = "graphnet"
     query_engine_class = MssqlQueryEngine
-    patient_join_column = "patient_id"
+    patient_join_column = "Patient_ID"
 
     patients = MappedTable(
-        source="Patients",
+        source="TRE.Patients",
         columns=dict(
-            date_of_birth=Column("date", source="date_of_birth"),
+            sex=Column("varchar", source="Sex"),
+            date_of_birth=Column("date", source="DateOfBirth"),
+            date_of_death=Column("date", source="DateOfDeath"),
+        ),
+    )
+
+    clinical_events = MappedTable(
+        source="TRE.ClinicalEvents",
+        columns=dict(
+            code=Column("varchar", source="CTV3Code"),
+            date=Column("datetime", source="ConsultationDate"),
+            numeric_value=Column("float", source="NumericValue"),
+        ),
+    )
```

Comment:
This makes me realise that we're treating the `clinical_events` table as a single table, however in v2 so far we've only defined the ctv3 part of it.  TPP is set up similarly, with 2 different tables for ctv3 and snomed, so we'll need to address how to deal with those for both backends.  I think we can merge this PR with the current setup and deal with that on another ticket

Reply:
In our database CTV3 and Snomed are all in the same table, but have been implemented in the backend separately just for consistency with TPP, but my preference would definitely be to implement it as a single entity

Reply:
:+1: Let's leave it as it is for now and we'll bear that in mind when adding the feature; I think it makes sense to treat them as one table from a user perspective